olive/systems/azureml/aml_workflow_runner.py

```diff
@@ -43,14 +43,17 @@ def main(raw_args=None):
     remote_cache_dir = workflow_artifacts_path / "cache"
     remote_output_dir = workflow_artifacts_path / "output"
 
+    print("The cache of this workflow is stored at: ", str(remote_cache_dir))
```

Comment:
I don't think it's useful here. this will just print the path where the datastore is mounted as. something like `/tmp/...`. I was suggesting adding the log to the aml system before it submits the job. like here https://github.com/microsoft/Olive/blob/213ac7ae5361351d6d7a7a5f5f26039355f16818/olive/systems/azureml/aml_system.py#L130

Reply:
If this is a relative path then it would be fine. Yeah but you are right I can add the log to aml system.